docs/en/faq/common-issues-in-deployment-and-development-phase.md

```diff
@@ -1 +1,255 @@
+
+### 7. Dynamically adjusting the Logging level of Spring Boot via Apollo
+
+You can refer to [spring-cloud-logger](https://github.com/ctripcorp/apollo) in the [apollo-use-cases](https://github.com/ctripcorp/apollo-use-cases/tree/master/spring-cloud-logger) project and [spring-boot-logger](https://github.com/ctripcorp/apollo-use-cases/tree/master/spring-boot-logger) code examples.
+
+### 8. Register Config Service and Admin Service to a separate Eureka Server
+
+Apollo comes with Eureka as an internal registry implementation by default, and there is generally no need to consider deploying a separate registry for Apollo.
+
+However, some companies already have a set of Eureka, if you want to register Apollo's Config Service and Admin Service to achieve unified management, you can follow the steps below.
+
+#### 1. Configure Config Service to not start the built-in Eureka Server
+
+##### 1.1 1.5.0 and above
+
+Configure apollo-configService with `apollo.eureka.server.enabled=false`, either through bootstrap.yml or the -D parameter.
+
+##### 1.2 versions before 1.5.0
+
+Modify [com.ctrip.framework.apollo.configservice.ConfigServiceApplication](https://github.com/apolloconfig/apollo/blob/master/apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/ConfigServiceApplication.java) , change `@EnableEurekaServer` to `@ EnableEurekaClient`
+
+```java
+@EnableEurekaClient
+@EnableAspectJAutoProxy
+@EnableAutoConfiguration // (exclude = EurekaClientConfigBean.class)
+@Configuration
+@EnableTransactionManagement
+@PropertySource(value = {"classpath:configservice.properties"})
+@ComponentScan(basePackageClasses = {ApolloCommonConfig.class,
+    ApolloBizConfig.class,
+    ConfigServiceApplication.class,
+    ApolloMetaServiceConfig.class})
+public class ConfigServiceApplication {
+  ...
+}
+```
+
+#### 2. Modify `eureka.service.url` in ApolloConfigDB.ServerConfig table to point to your own Eureka address
+
+For example, if your own Eureka service address is 1.1.1.1:8761 and 2.2.2.2:8761, then set eureka.service.url in ApolloConfigDB.ServerConfig table to
+
+```
+http://1.1.1.1:8761/eureka/,http://2.2.2.2:8761/eureka/
+```
+
+Note that changing the Eureka address only requires changing the `eureka.service.url` in the ApolloConfigDB.ServerConfig table, not the meta server address.
+
+> By default, the meta service and config service are deployed in the same JVM process, so the address of the meta service is the address of the config service, so you don't need to change the meta server address when you modify the Eureka address.
+
+### 9. Using `ConditionalOnProperty` in Spring Boot does not read the configuration
+
+The `@ConditionalOnProperty` feature is supported since version 0.10.0, see [Spring Boot integration method](en/usage/java-sdk-user-guide?id=_3213-spring-boot integration method (recommended))
```

Comment:
```suggestion
The `@ConditionalOnProperty` feature is supported since version 0.10.0, see [Spring Boot integration method](en/usage/java-sdk-user-guide?id=_3213-spring-boot-integration-methods-recommended)
```